csrc/cpu/comm/ccl.cpp

```diff
@@ -206,6 +527,7 @@ PYBIND11_MODULE(TORCH_EXTENSION_NAME, m)
     m.def("get_world_size", &get_world_size, "get world size");
     m.def("broadcast", &broadcast, "ccl broadcast");
     m.def("all_reduce", &all_reduce, "ccl all_reduce");
+    m.def("inference_allreduce", &inference_allreduce, "low latency all_reduce implementation");
```

Comment:
Please add some unit tests for this.

Reply:
@delock, thanks. 

@mrwyattii, can you please take a look at the unit test changes?